src/pydocstyle/utils.py

```diff
@@ -25,3 +25,16 @@ def pairwise(
     a, b = tee(iterable)
     _ = next(b, default_value)
     return zip_longest(a, b, fillvalue=default_value)
+
+
+def common_prefix_length(a: str, b: str) -> int:
+    """Return the length of the longest common prefix of a and b.
+
+    >>> common_prefix_length('abcd', 'abce')
+    3
+
+    """
+    for common, (ca, cb) in enumerate(zip(a, b)):
+        if ca != cb:
+            return common
+    return common + 1
```

Comment:
This function will crash with `UnboundLocalError` if either of the arguments is the empty string, which is probably not going to happen in this usage but nevertheless isn't very clean. My fault again on this.

Reply:
Done.